stigaview_static/main.py

```
@@ -30,42 +45,57 @@ def parse_args() -> argparse.Namespace:
+    parser.add_argument(
+        "-l", "--log-level", help="Log level", default="DEBUG", type=_log_level_type
+    )
     return parser.parse_args()
 
 
 def load_config(path: str) -> dict:
+    logging.info("Loading global config")
     if not os.path.exists(path):
-        sys.stderr.write(f"Config file not found: {path}\n")
+        logging.error(f"No such file: {path}")
         exit(3)
```

Pylint (3.3.7)

[refactor] Consider using 'sys.exit' instead

(R1722)